deploy/gallery/instructions.md

```diff
@@ -55,5 +57,16 @@ The instructions on this page will guide you through configuring the pipeline te
     | 8|EnvCode | \<environmentCode> | Same as used in the deployment steps|
     | 9|KeyVaultName | kvp<10-character-random-string>| Get the name from \<environmentCode>-pipeline-rg |
 
-5. All set to start the pipeline now. Press **Debug**.
+4. All set to start the pipeline now. Click **Debug**.
+
+# Verify Success
+After the pipeline has run, you should see an indication of Success for the pipeline in the Synapse Workspace. Also, you can find the geojson file that was generated using the following steps:
+
+1. Go to the resource group named **EnvironmentCode-data-rg**.
+2. In that resource group, go to the Storage Account whose name starts with **rawdata**.
+3. Click the **Containers** blade.
+4. Click the container with the name **EnvironmentCode-test-container**.
+5. Click the folder named **pool-geolocation**. You should see the **output.geojson** file there.
 
+# Cleanup
+After you have finished with using this sample, you can clean up resources by following the steps in the [Cleanup Script](https://github.com/Azure/Azure-Orbital-Analytics-Samples/tree/main/deploy#cleanup-script) section of the deployment Readme.
```

Comment:
We need to make sure what cleanup entails so that users who would like to retain workload do not end up accidentally deleting it because it takes a long time to deploy them and setup correctly.

Reply:
This section in Instructions.md that refers to Cleanup points back to the section in Readme.md that talks about Cleanup in more detail, including what gets changed and how to keep stuff if you want. That is, the link provided goes to the instructions in the Readme, NOT the script itself. So are there still changes needed to this section of Instructions.md?

Reply:
In that case, please disregard my comment.